Packages/com.unity.inputsystem/InputSystem/InputSettings.cs

```diff
@@ -518,6 +518,8 @@ public ReadOnlyArray<string> supportedDevices
             }
         }
 
+        public bool disableMouseMoveCompression { get; set; }
```

Comment:
Maybe this public property would benefit from a comment describing the impact of setting it to true/false or changing it during run-time?